weed/storage/volume_read_write.go

```diff
@@ -63,7 +64,11 @@ func (v *Volume) Destroy() (err error) {
 	return
 }
 
-func (v *Volume) writeNeedle(n *needle.Needle, fsync bool) (offset uint64, size uint32, isUnchanged bool, err error) {
+func (v *Volume) asyncRequestAppend(request *needle.AsyncRequest) {
+	v.asyncRequestsChan <- request
+}
+
+func (v *Volume) writeNeedleDecreated(n *needle.Needle, fsync bool) (offset uint64, size uint32, isUnchanged bool, err error) {
```

Comment:
"writeNeedleDecreated" you mean "writeNeedleDeprecated"?

Reply:
I use writeneedle2 instead, but not delete the old method, just rename it to avoid called by others

Reply:
yes, I mean writeNeedleDeprecated, I will update it, thanks